ci/linux_apisix_current_luarocks_runner.sh

```diff
@@ -33,8 +33,8 @@ script() {
     sudo rm -rf /usr/local/share/lua/5.1/apisix
 
     # install APISIX with local version
-    sudo luarocks install rockspec/apisix-master-0.rockspec --only-deps > build.log 2>&1 || (cat build.log && exit 1)
-    sudo luarocks make rockspec/apisix-master-0.rockspec > build.log 2>&1 || (cat build.log && exit 1)
+    sudo luarocks install rockspec/apisix-master-0.rockspec --only-deps | sudo tee build.log > /dev/null 2>&1 || (cat build.log && exit 1)
```

Comment:
We don't need sudo to write to `build.log`? So the original code's OK.

Reply:
The original redirects stdout into `build.log`, it isn't recommended for using `sudo`.

If it needs log into `build.log`, suggest using `tee` to make it.

Reply:
No, use no-sudo user is enough to write log to `build.log`.